CHANGES.rst

```diff
@@ -1,3 +1,16 @@
+5.0 (unreleased)
```

Comment:
please do not add sections to the changelog. This will just cause a guaranteed conflict, and anyway, the PR milestoned for the future cannot be merged until the branching for the earlier releases are done.

Reply:
Should I move this back to an earlier release in the Changelog, or leave it?